brut.j.util/src/main/java/brut/util/AaptManager.java

```diff
@@ -42,6 +42,8 @@ private static File getAppt(Integer version) throws BrutException {
         try {
             if (OSDetection.isMacOSX()) {
                 aaptBinary = Jar.getResourceAsFile("/prebuilt/" + aaptVersion + "/macosx/" + aaptVersion, AaptManager.class);
+            } else if(OSDetection.isAndroid()){
```

Comment:
So if a unix environment has this `/system/bin/linker64` file, it will pass the `isAndroid` check. Should the check for `IsAndroid` be at the end of this check?

Reply:
No. Android uses linux kernel. I have made sure that android will be detected as linux. Nomal linux platform won't has "/system/bin/linker64", they all uses glibc or uclibc/musl, instead of binoic. So if we find "/system/bin/linker64", we can think it must be android platform.